vnext/ExperimentalFeatures.props

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Could we do this in a separate change? I think we would also need to update the binary distribution project template to ensure that it works out of the box with Hermes enabled.

Reply:
Filed a bug to track this: https://github.com/microsoft/react-native-windows/issues/8201

Reply:
note that we can't turn hermes on for Debug builds of an app since it is not usable at the moment :-/